core/server/common/src/main/java/alluxio/cli/Format.java

```diff
@@ -55,12 +55,12 @@ private static boolean formatFolder(String name, String folder) throws IOExcepti
           failedToDelete = !ufs.deleteFile(childPath);
         }
         if (failedToDelete) {
-          LOG.info("Failed to delete {}", childPath);
+          LOG.error("Failed to delete {}", childPath);
```

Comment:
by our logging convention, `LOG.error` should only be used for fatal errors ... I believe you want `LOG.warn` here